modules/oe_theme_content_call_proposals/oe_theme_content_call_proposals.module

```diff
@@ -37,25 +37,20 @@ function oe_theme_content_call_proposals_node_view_alter(array &$build, EntityIn
   // Field labels are different in frontend and backend.
   // Here we override them, in full and teaser view mode.
   $label_overrides = [];
-  if (in_array($build['#view_mode'], ['full', 'teaser'])
-    && ($entity->get('oe_call_proposals_deadline')->count() > 1)) {
+  if ($entity->get('oe_call_proposals_deadline')->count() > 1) {
     $label_overrides['oe_call_proposals_deadline'] = t('Deadline dates');
   }
 
-  if ($build['#view_mode'] === 'full') {
-    $label_overrides += [
-      'oe_departments' => t('Departments'),
-      'oe_call_proposals_grants' => t('Results'),
-    ];
+  _oe_theme_helper_field_view_label_override($build, $entity, $label_overrides, ['full', 'teaser']);
```

Comment:
We are making changes out of scope, affecting other content types in an epic is a potential cause of issues. This ticket is about making teaser view mode for Consultation content type, that doesn't include changes on other CTs. I see the good intentions here, but this should be in a separate ticket & PR so it's well documented decision, team is aware and had a chance to evaluate and estimate it.

Reply:
After discussion with @ademarco we agreed to cherry-pick those changes to 508 ticket.